src/main/java/frc/trigon/robot/subsystems/shooter/Shooter.java

```diff
@@ -0,0 +1,63 @@
+package frc.trigon.robot.subsystems.shooter;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.trigon.robot.utilities.Conversions;
+
+public class Shooter extends SubsystemBase {
+
+    // With eager singleton initialization, any static variables/fields used in the 
+    // constructor must appear before the "INSTANCE" variable so that they are initialized 
+    // before the constructor is called when the "INSTANCE" variable initializes.
+    WPI_TalonFX masterMotor = ShooterConstants.MASTER_MOTOR;
+    private int ballCount = 0;
+
+    public void setTargetVelocity(double velocity) {
+        masterMotor.set(ControlMode.Velocity, velocity);
+    }
+
+    public double getTargetVelocity() {
+        if(masterMotor.getControlMode().equals(ControlMode.Velocity)) {
+            return masterMotor.getClosedLoopTarget();
+        }
+        return 0;
```

Comment:
I don't think just returning 0 is the best thing to do... if for whatever reason the control mode is not what it should be then you should probably have some kind of warning... ask @laviRZ what he suggests

Reply:
I think 0 is good. If you're in open loop the target is basically 0...

Reply:
Don't you think printing a warning would be good in addition? (If not, you can resolve the issue)